### ISSUE TITLE
--numpy cannot take 'all' or be passed multiple times

### ISSUE THREAD

**Issue by guyer** [conda/conda-build#1608]
`conda build --help` reports
```
  --numpy NUMPY_VER     Set the NumPy version used by conda build. Can be
                        passed multiple times to build against multiple
                        versions. Can be 'all' to build against all known
                        versions (['1.6', '1.7', '1.8', '1.9', '1.10',
                        '1.11'])
```
but `conda build --numpy all ...` reports
```
Traceback (most recent call last):
  File "/Users/guyer/anaconda/bin/conda-build", line 6, in <module>
    sys.exit(conda_build.cli.main_build.main())
  File "/Users/guyer/anaconda/lib/python2.7/site-packages/conda_build/cli/main_build.py", line 255, in main
    execute(sys.argv[1:])
  File "/Users/guyer/anaconda/lib/python2.7/site-packages/conda_build/cli/main_build.py", line 196, in execute
    config = Config(**args.__dict__)
  File "/Users/guyer/anaconda/lib/python2.7/site-packages/conda_build/config.py", line 47, in __init__
    self.set_keys(**kwargs)
  File "/Users/guyer/anaconda/lib/python2.7/site-packages/conda_build/config.py", line 81, in set_keys
    self.CONDA_NPY = int(self.CONDA_NPY.replace('.', '')) or None
ValueError: invalid literal for int() with base 10: 'all'
```
and `conda build  --numpy 1.9 --numpy 1.10 ...` generates
```
Traceback (most recent call last):
  File "/Users/guyer/anaconda/bin/conda-build", line 6, in <module>
    sys.exit(conda_build.cli.main_build.main())
  File "/Users/guyer/anaconda/lib/python2.7/site-packages/conda_build/cli/main_build.py", line 255, in main
    execute(sys.argv[1:])
  File "/Users/guyer/anaconda/lib/python2.7/site-packages/conda_build/cli/main_build.py", line 225, in execute
    set_language_env_vars(args, parser, config=config, execute=execute)
  File "/Users/guyer/anaconda/lib/python2.7/site-packages/conda_build/render.py", line 43, in set_language_env_vars
    execute(args, parser, config)
TypeError: execute() takes exactly 1 argument (3 given)
```

**Comment by msarahan** [comment]
Sorry, known issue. The help is wrong.  Replacement is coming in #1585

**Comment by msarahan** [comment]
help fix will be in 2.1.0.  Replacement will be in 3.0.0

**Comment by msarahan** [comment]
The new way of doing this is with variant configuration files.  The PR implementing these, #1585, has been merged.  The docs are in a PR at https://github.com/conda/conda-docs/pull/414

Closing this issue for now.  Please reopen it if you have further questions or issues.
